config/bidderinfo.go

```diff
@@ -125,6 +125,9 @@ type Syncer struct {
 	// SupportCORS identifies if CORS is supported for the user syncing endpoints.
 	SupportCORS *bool `yaml:"supportCors" mapstructure:"support_cors"`
 
+	// FormatOverride allows a bidder to override their callback type "b" for iframe, "i" for redirect
+	FormatOverride string `yaml:"formatOverride" mapstructure:"format_override"`
```

Comment:
Please update `validateSyncer` in `bidderinfo.go` to ensure the value is provided as either "", "b", or "i".

I recommend exposing `setuidSyncTypeIFrame` and `setuidSyncTypeRedirect` constants and use those instead of hardcoding the characters. Consider renaming `setuidSyncTypeIFrame` to something like `ResponseFormatIFrame`